email_reply_parser.gemspec

```diff
@@ -96,5 +96,8 @@ Gem::Specification.new do |s|
   ## Test files will be grabbed from the file list. Make sure the path glob
   ## matches what you actually use.
   s.test_files = s.files.select { |path| path =~ /^test\/.*_test\.rb/ }
+
```

Comment:
There is a comment block for add_development_dependency above, so maybe move this up there for neatness sake  :)